index.html

```diff
@@ -395,6 +395,25 @@ <h2>A project for first-time contributions</h2>
         </div>
         <!-- ________ Eugenio's card END ________  -->
 
+        <!-- ________ Judit's card START ________  -->
+        <div class="card">
+
+            <p class="name">Judit Csizmadiáné Pacsai</p>
+            <p class="contact"><i class="fa fa-github" aria-hidden="true"></i> <a href="https://github.com/jpacsai" target="_blank">@jpacsai</a></p>
+            <p class="about">I'm an office administrator from Hungary living in the UK for 11 years now. Fallen head over heels in love with coding, determined to get a job in the Web Development field next year. I love cats, Pentatonix and movies with Robert Downey Jr. <img alt="😀" class="imgs" src="data:image/png;base64,R0lGODlhDAAPAKIFAJh3AP/zxAAAANyuAP/gaP///wAAAAAAACH/
+                C05FVFNDQVBFMi4wAwEAAAAh+QQJAAAFACwAAAAADAAPAAADLVi63P6wjEmrDDjrGYj/H0cIIElwpJmeQweC4hu2BGDfnnjjrLvPLtlJUikWEgAh+QQJAAAFACwAAAAADAAPAAADLVi63M4jyllGuDjHQLr3GyF8I7GNJWpaXxu2IAvMdBfS9coReLzDpspk+HAkAAAh+QQJCgAFACwAAAAADAAPAAADLVi6vPMwlhGqvS+QzXkmQhcSWTiaJNWt3+qpQCxvnzynGmG/
```

Comment:
This image tag has an unusual source and appears to be broken. 
`<img alt="😀" class="imgs" src="data:image/png;base64,R0lGODlhDAAPAKIFAJh3AP/zxAAAANyuAP/gaP///wAAAAAAACH/C05FVFNDQVBFMi4wAwEAAAAh+QQJAAAFACwAAAAADAAPAAADLVi63P6wjEmrDDjrGYj/H0cIIElwpJmeQweC4hu2BGDfnnjjrLvPLtlJUikWEgAh+QQJAAAFACwAAAAADAAPAAADLVi63M4jyllGuDjHQLr3GyF8I7GNJWpaXxu2IAvMdBfS9coReLzDpspk+HAkAAAh+QQJCgAFACwAAAAADAAPAAADLVi6vPMwlhGqvS+QzXkmQhcSWTiaJNWt3+qpQCxvnzynGmG/uUtOkWBjSCQmAAAh+QQJFAAFACwAAAAADAAPAAADLViz3BowyhWIvZcSgTmhnBd+Q4Vh2pmVBOC+lvbCpDmvpvopTV/8wKBwSCwWEgAh+QQJCgAFACwAAAAADAAPAAADLVi6vPMwlhGqvS+QzXkmQhcSWTiaJNWt3+qpQCxvnzynGmG/uUtOkWBjSCQmAAAh+QQFAAAFACwAAAAADAAPAAADLVi63M4jyllGuDjHQLr3GyF8I7GNJWpaXxu2IAvMdBfS9coReLzDpspk+HAkAAA7">`
Can you please fix it before I can merge